src/utils/string.h

```diff
@@ -72,6 +73,7 @@ void replaceAll(std::string *str, const std::string& from,
     const std::string& to);
 std::string removeWhiteSpacesIfNeeded(std::string a);
 std::string parserSanitizer(std::string a);
+std::string log_escape_hex(std::string s);
```

Comment:
This should be removed, since a function by that name is no longer needed and there is no function definition.

Reply:
Oh, sorry - removed. Thanks.